desktop/containerd/index.md

```diff
@@ -16,8 +16,41 @@ image and file system management in the Docker Engine.
 > this feature in production environments as this feature may change or be
 > removed from future releases.
 
+## What is the containerd image store?
+
+containerd is a container runtime that manages the container lifecycle, and
+provides image and filesystem management. It's a low-level building block,
+designed to be integrated into other systems, such as Docker and Kubernetes.
+
+Docker Engine already uses containerd for container lifecycle management, which
+includes creating, starting, and stopping containers. This page describes the
+next step of containerd integration for Docker Engine: the image store.
```

Comment:
Maybe we can remove the whole "image store" wording and say : `image management`?